crates/ln-dlc-node/src/config.rs

```diff
@@ -1,9 +1,48 @@
+use std::time::Duration;
+
+use lightning::chain::chaininterface::ConfirmationTarget;
 use lightning::ln::channelmanager::MIN_CLTV_EXPIRY_DELTA;
 use lightning::util::config::ChannelConfig;
 use lightning::util::config::ChannelHandshakeConfig;
 use lightning::util::config::ChannelHandshakeLimits;
 use lightning::util::config::UserConfig;
 
+/// When handling the [`Event::HTLCIntercepted`], we may need to
+/// create a new channel with the recipient of the HTLC. If the
+/// payment is small enough (< 1000 sats), opening the channel will
+/// fail unless we provide more outbound liquidity.
+///
+/// This value defines the maximum channel amount between the coordinator and a user that opens a
+/// channel through an interceptable invoice. Channels that exceed this amount will be rejected.
+/// This value is completely arbitrary.
+///
+/// This constant only applies to the coordinator.
+pub const JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT_MAX: u64 = 200_000;
+
+/// The multiplier to be used by the coordinator to define the just in time channel liquidity
+///
+/// The liquidity provided by the trader will be multiplied with this value to defined the channel
+/// value.
+/// See `JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT_MAX` for the maximum channel value.
+pub const LIQUIDITY_MULTIPLIER: u64 = 2;
+
+/// Sats/vbyte rate for position
+///
+/// The coordinator and the app have to align on this to agree on the fees.
+pub const CONTRACT_TX_FEE_RATE: u64 = 4;
+
+/// The speed at which we want a transaction to confirm used for feerate estimation.
+///
+/// We set it to high priority because the channel funding transaction should be included fast.
+pub const CONFIRMATION_TARGET: ConfirmationTarget = ConfirmationTarget::HighPriority;
+
+/// When handling the [`Event::HTLCIntercepted`], the user might not be online right away. This
+/// could be because she is funding the wallet through another wallet. In order to give the user
+/// some time to open 10101 again we wait for a bit to see if we can establish a connection.
+///
+/// This constant specifies the amount of time we are willing to delay a payment.
+pub(crate) const HTLC_INTERCEPTED_CONNECTION_TIMEOUT: Duration = Duration::from_secs(30);
+
 pub fn app_config() -> UserConfig {
```

Comment:
🔧 Might be worth renaming this to `app_ldk_config` since we've lost the context of the module name.